src/components/images_uploading/UploadImageFile.js

```diff
@@ -31,6 +32,14 @@ class UploadImageFile extends Component {
       LocalApi.post("/images", formData).then(res => {
         console.log(res.statusText);
       });
+
+      //clear input field (reset to original state) after submitting the form

```

Comment:
After we create a new apartment, the data entered into the input fields will be cleared.

Reply:
It is better to clear the state when we know that data is successfully uploaded:
LocalApi.post(...).then(res => {
  ...
  this.setState({
        selectedFile: null,
        lot: 0,
        unitNumber: "",
        productDescription: ""
      });
}).

Reply:
Cool! I'll amend the code accordingly :)